src/common/NoResultsPage/index.js

```diff
@@ -3,10 +3,10 @@ import { NoResultsImage, Wrapper } from "./styled";
 import { Title } from "../Title";
 
 
-export const NoResultsPage = () => {
+export const NoResultsPage = ({query}) => {
   return (
     <Section>
-      <Title> Sorry, there are no results for “Muan” </Title>
+      <Title>{`Sorry, there are no results for "${query[0].toUpperCase() + query.slice(1)}"`}</Title>
```

Comment:
Consider making a function out of this and putting it in a separete file, because we will use it in MovieList and PersonList as well :)
```
const capitalize = (query) => query[0].toUpperCase() + query.slice(1);